ApplyToBecomeInternal/ApplyToBecome.Data/Services/AcademyConversionProjectRepository.cs

```diff
@@ -63,18 +63,14 @@ public async Task<ApiResponse<AcademyConversionProject>> UpdateProject(int id, U
 			return new ApiResponse<AcademyConversionProject>(response.StatusCode, project);
 		}
 
-		public Task<ApiResponse<List<string>>> GetAvailableStatuses()
+		public async Task<ApiResponse<List<string>>> GetAvailableStatuses()
 		{
-			return Task.FromResult(new ApiResponse<List<string>>(HttpStatusCode.OK, new List<string>
-			{
-				"Converter Pre-AO (C)",
-				"Pre Advisory Board",
-				"Active",
-				"Approved",
-				"Approved with Conditions",
-				"Deferred",
-				"Declined"
-			}));
+			HttpResponseMessage response = await _httpClient.GetAsync("v2/conversion-projects/statuses");
+
+			if (response.IsSuccessStatusCode is false)
+				return new ApiResponse<List<string>>(HttpStatusCode.NotFound, null);
```

Comment:
Would it be better to use the response.StatusCode here?
I suppose there's a chance that it is a different failure code.